modules/cluster-logging-collector-log-forward-examples.adoc

```diff
@@ -0,0 +1,385 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forward-examples_{context}"]
+= About forwarding logs to an external Elasticsearch instance
+
+A typical Log Forwarding configuration would be similar to the following examples.
+
+[id=cluster-logging-collector-log-forward-examples-es]
+== Forwarding logs to an external Elasticsearch instance
+
+The following example forwards all logs to a single external Elasticsearch instance:
+
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1alpha1"
+kind: "LogForwarding"
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+   - name: elasticsearch <1>
+     type: "elasticsearch" <2>
+     url: elasticsearch.example.com:9200 <3> 
+  pipelines:
+   - name: container-logs <4>
+     inputRefs: <5>
+     - infrastructure
+     - application
+     - audit
+     outputRefs:
+     - elasticsearch <6>
+----
+<1> Specify a name for the output.
+<2> Specify the `elasticsearch` type.
+<3> Specify the URL and port of the external Elasticsearch instance.
+<4> Optional. Specify a name for the pipeline.
+<5> Specify which log types should be forwarded.
+<6> Specify the output to use for forwarding the logs.
+
+[id=cluster-logging-collector-log-forward-examples-es-2]
+== Forwarding logs to the internal Elasticsearch instance and an external Elasticsearch instance
+
+The following example forwards all logs to a single external Elasticsearch instance and to the OpenShift Container Platform Elasticsearch instance:
+
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1alpha1"
+kind: "LogForwarding"
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+   - name: elasticsearch <1>
+     type: "elasticsearch" <2>
+     url: elasticsearch.example.com:9200 <3>
```

Comment:
`url: http://elasticsearch.example.com:9200`

Reply:
Removed this topic.